github-templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+(brief description / PR title)
+
+Status: **Ready for Review** or **Open for Visibility**
+Reviewers: <reviewers here>
+
+JIRA: <link to applicable JIRA ticket>
+
+## Changes
+- <changes here>
+
+## Todos:
+- [ ] +1
+- [ ] Passes tests
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] <Other TODOs>
+
+### Feedback:
+_none so far_
```

Comment:
On CST we've used it as a TL;DR section for collecting the feedback from inline comments, and use checkboxes to indicate that the suggested changes have been completed. 

It gives future viewers of the PR a little more context of how the PR changed, and updates reviewers if they arrive late to the party.